featurebyte/__init__.py

```diff
@@ -126,6 +126,26 @@ def get_active_profile() -> Profile:
         If no profile is found in configuration file
     """
     conf = Configurations()
+
+    # check if we are in DataBricks environment and valid secrets are present create a profile automatically
+    db_utils = globals().get("dbutils")
+    if db_utils:
+        databricks_auto_profile_name = "databricks-auto-profile"
+        if conf.default_profile_name != databricks_auto_profile_name:
+            try:
+                api_url = db_utils.secrets.get(scope="featurebyte", key="api-url")
+                api_token = db_utils.secrets.get(scope="featurebyte", key="api-token")
+                register_profile(
+                    profile_name=databricks_auto_profile_name,
+                    api_url=api_url,
+                    api_token=api_token,
+                    make_default=True,
+                )
+            except Exception:  # pylint: disable=broad-except
```

Comment:
If possible would prefer to make the exception more specific. Or at least move `register_profile` out of the try block if we only expect `db_utils.secrets.get` to fail.

Reply:
The exception will be using a class that is not available in non databricks env so I will do the latter